scripts/init.js

```diff
@@ -68,21 +62,28 @@ module.exports = function(appPath, appName, verbose, originalDirectory) {
     }
   });
 
-  // Run another npm install for react and react-dom
+  // Run npm installs for react and react-dom
   console.log('Installing react and react-dom from npm...');
   console.log();
   // TODO: having to do two npm installs is bad, can we avoid it?
-  var args = [
-    'install',
-    verbose && '--verbose'
-  ].filter(function(e) { return e; });
-  var proc = spawn('npm', args, {stdio: 'inherit'});
-  proc.on('close', function (code) {
-    if (code !== 0) {
+
+  var installFailed;
+  ['react', 'react-dom', 'react-test-renderer'].forEach(function (pkg) {
+    var args = [
+      'install',
+      pkg,
+      pkg === 'react-test-renderer' ? '--save-dev' : '--save',
+      verbose && '--verbose'
+    ].filter(function(e) { return e; });
+    var result = spawnSync('npm', args, {stdio: 'inherit'});
```

Comment:
These should not be three separate spawns.
Let's just use `--save` for all of them, and do it as one spawn.